### PR TITLE
Sharing some learnings in this PR

### PR DESCRIPTION
**Pull request by bureado**
This is great work! I wanted to contribute a few learnings from my experience in open source go-to-market efforts. I'm sending you some suggestions to elevate the "entry barriers" and "business model" sections, and adding some resources. I hope this is useful.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,19 +138,26 @@ Using standard code conventions and infrastructure lowers the barrier for people
...
+  - [x] When appropriate, consider [talking about your project publicly](https://github.com/vmbrasseur/Public_Speaking)
+    - [x] Review the [Writing and performing a great conference presentation](https://archive.org/details/dodseattle2019-presentation-training) talk
 
 * Specific for Apps / Products
   - [x] Submit project to [Product Hunt](https://www.producthunt.com)
 
-To maintain communication with existing users, use a newsletter. Consider using [mailchimp](https://mailchimp.com) for this.
+To maintain communication with existing users, use a newsletter. Consider using [mailchimp](https://mailchimp.com) for this. Be aware that once communication becomes more fluid, you might create support expectations so it becomes critical to [understand open source support systems](https://nadiaeghbal.com/user-support).
+
+## 🤑 Business model
```
Comment (amilajack) [inline README.md:150]: I think we should leave the title as `Funding`. I think the term `Business Model` may make people think that we are suggesting they treat their project like a business, which isn't our intent (maybe we can create a different set of advice for people who want to do that).

File: README.md [hunk 2]
```diff
@@ -95,9 +97,9 @@ Make your project unique and memorable (most people only focus on technical aspe
     * Flow’s is yellow and gray
     * Make sure your docs and website follow this color theme
 - [x] Further reading
-  * If you want to learn more about marketing, see [LisaDziuba/Marketing-for-Engineers](https://github.com/LisaDziuba/Marketing-for-Engineers)
+  * If you want to learn more about marketing, see [LisaDziuba/Marketing-for-Engineers](https://github.com/LisaDziuba/Marketing-for-Engineers) and/or reach out to some open source marketing leaders such as [@kamcmahon](https://twitter.com/kamcmahon). [@jenlankford](https://twitter.com/jenlankford), [@amanda_katona](https://twitter.com/amanda_katona), [@FunnelFiasco](https://twitter.com/funnelfiasco) or [@Jen_L_Gray](https://twitter.com/jen_l_gray).
 
-## ✈️ Onboarding Methods
+## ✈️ Entry Barriers
```
Comment (amilajack) [inline README.md:103]: I think we should change the title to `Onboarding Users`. The overall section intent of the section is to convey that.

File: README.md [hunk 3]
```diff
@@ -45,6 +45,7 @@ Figure out the value your project brings to your users. What does it provide the
 
 * Recommendations
   * A value proposition can have multiple propositions
+  * [Writing down a market positioning statement](http://blog.ecornell.com/how-to-write-market-positioning-statements/) or statements can help with clarity, business goals, marketing and more
```
Comment (amilajack) [inline README.md:48]:
```suggestion
  * [Figure out who your target audience is](http://blog.ecornell.com/how-to-write-market-positioning-statements/)
```

**Review by amilajack — CHANGES_REQUESTED** [review]
These are awesome changes! Thank you so much for the contributions 🎉 I left some small changes to make before this is merged

**Comment by bureado** [comment]
Cool! I [added a few commits](https://github.com/bureado/project-checklist/commits/patch-1) earlier today in case you want to look into those as well.

File: README.md [hunk 4]
```diff
@@ -139,6 +139,7 @@ Using standard code conventions and infrastructure lowers the barrier for people
   - [x] Post on [twitter](https://twitter.com) and upload images of project
   - [x] Post to corresponding reddit programming language community [/r/javascript](https://www.reddit.com/r/javascript), [/r/python](https://www.reddit.com/r/python), etc
   - [x] When appropriate, consider [talking about your project publicly](https://github.com/vmbrasseur/Public_Speaking)
+    - [x] Review the [Writing and performing a great conference presentation](https://archive.org/details/dodseattle2019-presentation-training) talk
```
Comment (amilajack) [inline README.md:143]: I think we should add videos and lengthy readings to another section. Can you please add this to a new file called `Resources.md` with the following titles: `Talks` and `Readings` as h2 headers.

File: README.md [hunk 5]
```diff
@@ -107,6 +110,14 @@ In general, here's how this works: say you have library `y` that competes with p
   * [Figma](http://figma.com), a vector graphics editor, gained users by allowing [Sketch](http://sketchapp.com) users to import Sketch files into Figma
   * [Laravel](http://laravel.com) onboarded many [Rails](https://rubyonrails.org) users by providing terminology and conventions that were similar to [Rails](https://rubyonrails.org)
 
+In general, you can't expect your users to _engage_ in your project if the barriers are as high for them as they were for you as the original developer. Here are some additional ideas:
```
Comment (amilajack) [inline README.md:113]: > _engage_

This is usually a term that is common among non-technical people. Can we change this to a word that is more commonly used? How about `use`?

**Review by amilajack — CHANGES_REQUESTED** [review]

**Pull request closed without merging**